### ISSUE TITLE
mismatching score: lcoal installed spliceAI model VS spliceAI Look Up

### ISSUE THREAD

**Issue by woooh324** [broadinstitute/SpliceAI-lookup#26]
Hi,
First of all, thank you for inventing such an useful API.

I've run the spliceAI model locally to check spliceAI scores of larger InDels whose precomputed scores are not provided.
Most of the scores were matched with spliceAI Look Up, but some of them were not.

For instance, the score for del in ABCC9 is as follows
[ABCC9, chr12-21941195-CTTACTCCGATGACAG-C](https://spliceailookup.broadinstitute.org/#variant=chr12-21941195-CTTACTCCGATGACAG-C&hg=38&distance=100&mask=0&precomputed=0)

However, local spliceAI model outputs nothing for the variant. There should be spliceAI annotation if local spliceAI model predicted something.
![image](https://user-images.githubusercontent.com/108379939/182031187-7812c657-075d-43e2-8cc1-bb0713d8ec11.png)

Is there any possible reason for this situation?

I'm using
- spliceAI v1.3.1
- tensorflow v1.15.0
- keras v2.3.1
- h5py v2.10.0
- hg38
- D == 100
- raw score
for both model and API.

Thank you

**Comment by bw2** [comment]
@woooh324 this might be because the API uses a different version of gencode than what you're using locally.

**Comment by bw2** [comment]
You can find the annotation files used by the API @ https://github.com/broadinstitute/SpliceAI-lookup/tree/master/annotations
